src/PrestaShopBundle/Command/ModuleCommand.php

```diff
@@ -183,9 +195,15 @@ protected function executeConfigureModuleAction($moduleName, $file = null)
         );
     }
 
-    protected function executeGenericModuleAction($action, $moduleName)
+    protected function executeGenericModuleAction($action, $moduleName, $options = null)
```

Comment:
I'm wondering ... what if the default value of $options is an empty array ? This make the function easier as we can remove the if/else statment : WDYT ?

Reply:
As the call is not he same (or we need to edit all actions to have the $options params for nothing), you still need to make an if statement or you will have a second parameters send to action that expect only one.